torch/_dynamo/allowed_functions.py

```diff
@@ -252,7 +252,10 @@ def _builtin_function_ids():
 def _numpy_function_ids():
     rv = dict()
     if HAS_NUMPY:
-        for mod in (np, np.random):
+        modules = (
+            (np, np.random, np.linalg) if HAS_NUMPY_TORCH_INTEROP else (np, np.random)
```

Comment:
Not sure about the implication of supporting np.linalg functions when `HAS_NUMPY=True and HAS_NUMPY_TORCH_INTEROP=False`, so being conservative here.

Reply:
np.fft, too?

Reply:
yep, but see the OP

Reply:
Yeah, I'd try increasing modules unconditionally